tests/seccomp.rs

```diff
@@ -0,0 +1,34 @@
+#[cfg(target_os = "linux")]
+use std::ffi::CString;
+
+#[cfg(target_os = "linux")]
+use birdcage::{Birdcage, Sandbox};
+
+#[cfg(target_os = "linux")]
+fn main() {
+    // Activate our sandbox.
+    Birdcage::new().lock().unwrap();
+
+    // Ensure `chdir` is allowed.
+    let root_path = CString::new("/").unwrap();
+    let result = unsafe { libc::chdir(root_path.as_ptr()) };
+    assert_eq!(result, 0);
+
+    // Ensure `unshare` is always blocked.
+    let result = unsafe { libc::unshare(libc::CLONE_NEWUSER) };
+    assert_eq!(result, -1);
+
+    // Ensure `clone` is blocked with `CLONE_NEWUSER`.
+    let stack = unsafe { libc::malloc(4096) };
+    let flags = libc::CLONE_NEWUSER as libc::c_ulong;
+    let result = unsafe { libc::syscall(libc::SYS_clone, flags, stack) };
+    assert_eq!(result, -1);
+
+    // Ensure `clone` is allowed without flags.
+    let flags = 0;
+    let result = unsafe { libc::syscall(libc::SYS_clone, flags, stack) };
+    assert!(result > 0);
```

Comment:
The child crashes... Maybe because of this `assert!()`? I expect the syscall returns `0` in the child. Or maybe because of some other issue.

```
Stack trace of thread 4958:
#0  0x0000aaaad88392e4 n/a (/app/target/debug/deps/seccomp-72d3b96661912877 + 0x92e4)
#1  0x0000aaaad88392a8 n/a (/app/target/debug/deps/seccomp-72d3b96661912877 + 0x92a8)
#2  0x0000ffffbc6974cc n/a (/usr/lib/aarch64-linux-gnu/libc.so.6 + 0x274cc)
#3  0x0000aaaad8838e70 n/a (/app/target/debug/deps/seccomp-72d3b96661912877 + 0x8e70)
ELF object binary architecture: AARCH64
```

To be clear, it crashes with no output. And the test passes because it only checks the exit status of the parent. The stack trace above is from `systemd-coredump`.

Reply:
Decided to just remove the test. `Command::new` tests this anyway.